_sass/_styles.scss

```diff
@@ -195,7 +196,11 @@ Does not work on round things. */
 
 #fall-competition-2015 {
   background-image: url(../assets/images/gallery-photos/fall-comp-2015-5.jpg);
-  background-position: 0 60%;
+  background-position: 0 70%;
+}
+
+#spring-competition-2015 {
```

Comment:
After looking at the load times for the images we have (and imagining the pain of writing raw HTML every. single. time. we want to add an image to the gallery, I definitely want to host the images on Google Drive. Doing so requires interacting with the Google Drive API.

Good point about the styles. I'm hoping to have a purely programmatic way of displaying images from Google Drive in the gallery, so pre-defining styles by image category is probably not a good idea. So those CSS rules will probably change soon, but I'll merge the request 👍